src/integration/pages/storyPage/canonicalIncludeTests.js

```diff
@@ -3,7 +3,7 @@ export default () => {
     describe('IDT2', () => {
       it('I can see a "dataPic"', () => {
         const scriptEl = document.querySelector(
-          'script[src="https://news.test.files.bbci.co.uk/include/idt2/static/js/dataPic.8f264bc5.js"]',
+          'script[src*="https://news.test.files.bbci.co.uk/include/idt2/static/js/dataPic"]',
```

Comment:
Interesting, I tried this exact thing and it didn't work locally for me. Ah well, this is better 🤷